src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/LgtmScore.java

```diff
@@ -16,71 +16,65 @@
 import java.util.EnumMap;
 
 /**
- * The score shows if and how a project addresses issues reported by LGTM.
- * The score is based on the following features
+ * The score shows if and how a project addresses issues reported by LGTM. The score is based on the
+ * following features
  * <ul>
- *   <li>{@link com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures#USES_LGTM}</li>
- *   <li>{@link com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures#WORST_LGTM_GRADE}</li>
+ * <li>{@link OssFeatures#USES_LGTM_CHECKS}</li>
+ * <li>{@link OssFeatures#WORST_LGTM_GRADE}</li>
  * </ul>
  */
 public class LgtmScore extends FeatureBasedScore {
 
-  private static final double LGTM_USAGE_POINTS = 2.0;
+  /**
+   * Defines how the score value is increased if a project uses LGTM checks for commits.
+   */
+  private static final double LGTM_CHECKS_POINTS = 4.0;
 
+  /**
+   * Defines how LGTM grades contribute to the score value.
+   */
   private static final EnumMap<LgtmGrade, Double> GRADE_TO_POINTS = new EnumMap<>(LgtmGrade.class);
 
   static {
-    GRADE_TO_POINTS.put(LgtmGrade.A_PLUS, 8.0);
-    GRADE_TO_POINTS.put(LgtmGrade.A, 8.0);
-    GRADE_TO_POINTS.put(LgtmGrade.B, 6.0);
-    GRADE_TO_POINTS.put(LgtmGrade.C, 4.0);
+    GRADE_TO_POINTS.put(LgtmGrade.A_PLUS, 6.0);
```

Comment:
Is a static ENUM better here?

Reply:
I am not sure about it. We need a mapping from `LgtmGrade` to a double, and a map looks like a good option.